scripts/setup_fullnode.sh

```diff
@@ -367,7 +367,7 @@ function main() {
     # Interactive setup phase
     read -p "Do you want to monitor the status of the node [Y/N] (Default: N)? " wantmonitor
     if [ "${_AUTO_UPDATE_}X" != "YX" ];then
-        read -p "Do you want to auto update the node [Y/N] (Default: Y)? " _AUTO_UPDATE_
+        read -p "Do you want to auto update the node [Y/N] (Default: N)? " _AUTO_UPDATE_
```

Comment:
the code at 
scripts/setup_fullnode.sh:466

```
if _AUTO_UPDATE_ == empty string {
   startAutoUpdate 
}
```

maybe scripts/setup_fullnode.sh:466 need changed too

Reply:
@huangzhiran changed last if. Please review

Reply:
@xianhuawei Can you review?